src/knownnodes.py

```diff
@@ -61,8 +61,8 @@ def json_deserialize_knownnodes(source):
         knownNodes[node['stream']][peer] = info
 
         if (
-            not (knownNodesActual or info.get('self')) and
-            peer not in DEFAULT_NODES
+                not (knownNodesActual or info.get('self')) and
+                peer not in DEFAULT_NODES
```

Comment:
what's wrong here?

Reply:
Pylint: wrong hanging indentation before a block.

Reply:
I insist. You shouldn't follow pylint bugs. This is the pylint bug: https://github.com/PyCQA/pylint/issues/289

Reply:
Okay.